test_runner/regress/test_combocid.py

```diff
@@ -0,0 +1,37 @@
+from fixtures.neon_fixtures import NeonEnvBuilder
+
+
+def test_combocid(neon_env_builder: NeonEnvBuilder):
```

Comment:
@knizhnik: unfortunately, this test also passes with the current code. It's added because existing test doesn't catch anything